src/domain/registration/notEligible/NotEligibleUtils.ts

```diff
@@ -0,0 +1,40 @@
+import { newMoment } from '../../../common/time/utils';
+import { SUPPORTED_START_BIRTHDATE } from '../../../common/time/TimeConstants';
+import { RegistrationFormValues } from '../types/RegistrationTypes';
+
+/** isBirthdateEligible()
+ * Check if child is eligible for participation.
+ *
+ * Only children born in 2020 is eligible for this service.
+ * During development we allow dates in 2019.
+ * @param {string} birthdate in YYYY-MM-DD format.
+ * @returns {boolean}
+ */
+const isBirthdateEligible = (value: string) => {
+  const inputMoment = newMoment(value, 'YYYY-MM-DD');
+  const supportedStart = newMoment(SUPPORTED_START_BIRTHDATE);
+
+  if (!inputMoment.isValid()) return false;
+
+  if (inputMoment.isBefore(supportedStart)) {
+    return false;
+  }
+  return true;
+};
+
+const isCityEligible = (city: string) => {
+  let eligibleCities = process.env.REACT_APP_ELIGIBLE_CITIES;
```

Comment:
make a getSupportedCities function => const func = () => process.env.VAR to array of lowercase string || ['helsinki']